src/java.desktop/share/classes/com/sun/java/swing/plaf/motif/resources/motif_it.properties

```diff
@@ -55,7 +55,7 @@ FileChooser.pathLabel.textAndMnemonic=&Percorso o nome cartella:
 FileChooser.filterLabel.textAndMnemonic=Filt&ro
 FileChooser.foldersLabel.textAndMnemonic=Car&telle
 FileChooser.filesLabel.textAndMnemonic=F&ile
-FileChooser.enterFileNameLabel.textAndMnemonic=Immettere il &nome file: 
+FileChooser.enterFileNameLabel.textAndMnemonic=Immettere il &nome file:\u0020
```

Comment:
Unused, but most have the space included (except for en, oddly). Put a space in en?